### PR TITLE
Translate kustomize task page into bahasa indonesia

### PR DESCRIPTION
**Pull request by girikuncoro**
Part of #22296 , localizing `/docs/tasks/manage-kubernetes-objects/kustomization`

/language id
/cc @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 1]
```diff
@@ -0,0 +1,841 @@
...
+        - start
+        - --host
+        - dev-my-nginx-001
+        image: nginx
+        name: my-nginx
+```
+
+## Base dan Overlay
+
+Kustomize memiliki konsep **base** and **overlay**. **base** merupakan direktori dengan `kustomization.yaml`, yang berisi
```
Comment (ariscahyadi) [inline content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:660]: missed?
```suggestion
Kustomize memiliki konsep **base** dan **overlay**. **base** merupakan direktori dengan `kustomization.yaml`, yang berisi
```

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 2]
```diff
@@ -0,0 +1,841 @@
+---
+title: Mengelola Objek Kubernetes secara Deklaratif menggunakan Kustomize
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+[Kustomize](https://github.com/kubernetes-sigs/kustomize) merupakan sebuah alat
+untuk mengubah (_customize_) objek Kubernetes melalui sebuah berkas [berkas kustomization](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:10]:
```suggestion
untuk melakukan kustomisasi (_customization_) objek Kubernetes melalui sebuah berkas [berkas kustomization](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization).
```

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 3]
```diff
@@ -0,0 +1,841 @@
...
+
+## Gambaran Umum Kustomize
+
+Kustomize adalah sebuah alat untuk mengubah-ubah konfigurasi Kubernetes. Untuk mengelola berkas-berkas konfigurasi, kustomize memiliki fitur di bawah ini:
+
+* men-_generate_ sumber daya dari sumber lain
+* mengatur _field_ dari berbagai sumber daya yang bersinggungan
+* mengkomposisikan dan mengubah sekelompok sumber daya
+
+### Men-generate Sumber Daya
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:46]: sure I will use "membangkitkan" and introduce this word with parentheses

**Comment by girikuncoro** [comment]
@ariscahyadi ah I forgot "kustomisasi" is formal indonesian word, will address this

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 4]
```diff
@@ -0,0 +1,841 @@
...
+- deployment.yaml
+- service.yaml
+EOF
+```
+
+Sumber daya dari `kubectl kustomize ./` berisi kedua objek Deployment dan Service.
+
+#### Melakukan Perubahan
+
+_Patch_ dapat digunakan untuk menerapkan berbagai macam perubahan pada sumber daya. Kustomize mendukung berbagai mekanisme _patching_ yang berbeda melalui `patchesStrategicMerge` dan `patchesJson6902`. `patchesStrategicMerge` adalah daftar dari yang berisi _path_ berkas. Setiap berkas akan dioperasikan dengan cara [strategic merge patch](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md). Nama di dalam _patch_ harus sesuai dengan nama sumber daya yang telah dimuat. Kami menyarankan _patch-patch_ kecil yang hanya melakukan satu hal saja.
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md]: don't see anything different here from suggested change

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 5]
```diff
@@ -0,0 +1,841 @@
...
+        image: nginx
+        name: my-nginx
+```
+
+## Base dan Overlay
+
+Kustomize memiliki konsep **base** and **overlay**. **base** merupakan direktori dengan `kustomization.yaml`, yang berisi
+sekumpulan sumber daya dan perubahan yang terkait. **base** dapat berupa direktori lokal maupun direktori dari repo _remote_,
+asalkan berkas `kustomization.yaml` ada di dalamnya. **overlay** merupakan direktori dengan `kustomization.yaml` yang merujuk pada
+direktori kustomization lainnya sebagai **base**-nya. **base** tidak memiliki informasi tentang **overlay**. dan dapat digunakan pada beberapa **overlay** sekaligus.
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:663]: the suggested period seems incorrect

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 6]
```diff
@@ -0,0 +1,841 @@
...
+| crds                  | []string                                                                                                     | setiap entri di dalam daftar ini harus diselesaikan ke sebuah berkas definisi OpenAPI untuk tipe Kubernetes |
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+* [Kustomize](https://github.com/kubernetes-sigs/kustomize)
+* [Buku Kubectl](https://kubectl.docs.kubernetes.io)
+* [Rujukan Perintah Kubectl](/docs/reference/generated/kubectl/kubectl/)
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:838]: we don't have this yet, so I leave it as english reference

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 7]
```diff
@@ -0,0 +1,841 @@
...
+* [Rujukan API Kubernetes](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:839]: we don't have this yet, so I leave it as english reference

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 8]
```diff
@@ -0,0 +1,841 @@
...
+- deployment.yaml
+- service.yaml
+EOF
+```
+
+Sumber daya dari `kubectl kustomize ./` berisi kedua objek Deployment dan Service.
+
+#### Melakukan Kustomisasi
+
+_Patch_ dapat digunakan untuk menerapkan berbagai macam kustomisasi pada sumber daya. Kustomize mendukung berbagai mekanisme _patching_ yang berbeda melalui `patchesStrategicMerge` dan `patchesJson6902`. `patchesStrategicMerge` adalah daftar dari yang berisi _path_ berkas. Setiap berkas akan dioperasikan dengan cara [strategic merge patch](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md). Nama di dalam _patch_ harus sesuai dengan nama sumber daya yang telah dimuat. Kami menyarankan _patch-patch_ kecil yang hanya melakukan satu hal saja.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:349]: I don't know why it is not shown in the previous comment. The issue is in "... adalah daftar dari yang berisi _path_ berkas." 
Can we make into " ... adalah daftar yang berisi tentang _path_ berkas."?

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:838]: I thought there is a fallback mechanism to English Docs, I think it was mentioned by @sftim .

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md:839]: I thought there is a fallback mechanism to English Docs, I think it was mentioned by @sftim .

File: content/id/docs/tasks/manage-kubernetes-objects/kustomization.md (hunk 8 above)
Comment (girikuncoro) [reply content/id/docs/tasks/manage-kubernetes-objects/kustomization.md]: ok done in 920197b

**Comment by girikuncoro** [comment]
> I thought there is a fallback mechanism to English Docs, I think it was mentioned by @sftim .

@ariscahyadi you are right with this, I've added `/id` prefix in 920197b and check in the preview netlify, works beautifully (fallback to EN pages)

**Comment by sftim** [comment]
> I thought there is a fallback mechanism to English Docs

It's not a fallback mechanism. But there _might_ in future be a way to automatically replace a link to the upstream English doc with a link to its localized replacement.

**Pull request merged**